qiita_pet/templates/study_ajax/prep_summary.html

```diff
@@ -242,9 +242,8 @@
     if(confirm("Are you sure you want to delete prep template " + prep_id + "?")) {
       // Perform the AJAX call to delete
       $.ajax({
-        url: '/prep_template/',
+        url: '/prep_template/?prep-template-id=' + prep_id,
```

Comment:
Perhaps but that's what the full system expects at this point. If you feel strongly about this, could you open an issue?